Gems/MiniAudio/Code/Source/Clients/MiniAudioListenerComponentConfig.cpp

```diff
@@ -20,6 +20,21 @@ namespace MiniAudio
                 ->Version(1)
                 ->Field("Follow Entity", &MiniAudioListenerComponentConfig::m_followEntity)
                 ->Field("Listener Index", &MiniAudioListenerComponentConfig::m_listenerIndex)
+                ->Field("Inner Cone Angle (Degrees)", &MiniAudioListenerComponentConfig::m_innerAngleInDegrees)
+                    ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                    ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                    ->Attribute(AZ::Edit::Attributes::Max, 360.0f)
+                    ->Attribute(AZ::Edit::Attributes::Suffix, " degrees")
+                ->Field("Outer Cone Angle (Degrees)", &MiniAudioListenerComponentConfig::m_outerAngleInDegrees)
+                    ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                    ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                    ->Attribute(AZ::Edit::Attributes::Max, 360.0f)
+                    ->Attribute(AZ::Edit::Attributes::Suffix, " degrees")
+                ->Field("Outer Volume (%)", &MiniAudioListenerComponentConfig::m_outerVolume)
+                    ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                    ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                    ->Attribute(AZ::Edit::Attributes::Max, 100.0f)
+                    ->Attribute(AZ::Edit::Attributes::Suffix, " %")
```

Comment:
The edit context attributes should not be part of the serialize context declaration of the class
```suggestion
                ->Field("Inner Cone Angle", &MiniAudioListenerComponentConfig::m_innerAngleInDegrees)
                ->Field("Outer Cone Angle", &MiniAudioListenerComponentConfig::m_outerAngleInDegrees)
                ->Field("Outer Volume", &MiniAudioListenerComponentConfig::m_outerVolume)
```